docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
I refactored the two docker files into docker+bootstrap files here:  https://github.com/brycem/PowerShell/tree/docker/docker/release - They work fine off a local share, but because of the way Docker build context works, you lose the user simplicity of doing 'docker build https://raw.githubusercontent.com/brycem/PowerShell/docker/docker/release/windowsservercore/Dockerfile' like you could with the current self-contained dockerfile format...  I'd recommend refactoring the Linux files to inline the code for the more flexible usability.